lib/src/index.ts

```diff
@@ -237,3 +237,5 @@ export { default as NxTransferList, Props as NxTransferListProps, DataItem as Nx
 
 export { default as NxStatefulTransferList, Props as NxStatefulTransferListProps }
   from './components/NxTransferList/stateful/NxStatefulTransferList';
+
+export { NxDivider } from './components/NxDivider/NxDivider';
```

Comment:
I'm surprised it isn't causing compilation errors to have both this and the copy is SimpleCompnonents defined and exported.  Shouldn't that cause a naming collision?

Reply:
> I'm surprised it isn't causing compilation errors to have both this and the copy is SimpleCompnonents defined and exported. Shouldn't that cause a naming collision? Regardless, this should be deleted.

Not sure why I didn't get any compilation errors. This should've been caught easily.
Removed. Check e1f79dfe5abbe6861ef08f7b224747b2b200e2dd